src/main/openapi/ds-discover-openapi_v1.yaml

```diff
@@ -558,6 +558,99 @@ paths:
                   Output compatible with [Solr CSV response](https://solr.apache.org/guide/8_10/response-writers.html#csv-response-writer)
                 type: string
 
+
+  /solr/{collection}/suggest:
+    get:
+      tags:
+        - '${project.name}'
+      summary: 'Perform a Solr suggest in the stated collection and dictionary'
+      operationId: solrSuggest
+      parameters:
+
+        - name: collection
+          in: path
+          description: 'The ID of the Solr collection to search. Available collections can be requested from /solr/admin/collections'
+          required: true
+          schema:
+            type: string
+            default: 'ds'
+            example: 'ds'
+
+
+        - name: suggestDictionary
+          in: query
+          description: 'The suggest dictionary for use. The dictionary must be defined in solr.'
+          required: true
+          schema:
+            type: string
+            default: 'dr_title_suggest'
+            example: 'dr_title_suggest'
+
+        - name: suggestQuery
```

Comment:
Please use the Solr parameters instead of `suggestDictionary` and `suggestQuery`: The idea is that `ds-discover` is interchangeable with a standard Solr.

Reply:
Correct. Big mistake.